src/internal/channel.js

```diff
@@ -13,9 +13,21 @@ export function emitter() {
     return () => remove(subscribers, sub)
   }
 
+  function emitAll(items) {
+    for (var i = 0, len = items.length; i < len; i++) {
+      emit(items[i])
+    }
+  }
+
   function emit(item) {
+    if (item === undefined) {
+      return
+    }
+    if (Array.isArray(item)) {
```

Comment:
thats a breaking change as even without `processAction` option is theoretically possible to dispatch an array and with that change it would suddenly start to emit each item in it instead of the original item